FFL-900 Addresing `feature/flags` PR feedback (#1113)
DataDog/dd-sdk-reactnative: packages/react-native-openfeature/README.md

## Setup

> **Note**: This package is an integration for the [OpenFeature React SDK][1]. Before starting using Datadog Feature Flags, install and set up the core [`@datadog/mobile-react-native`][3] SDK.

Reviewer: ```suggestion
**Note**: This package is an integration for the [OpenFeature React SDK][1]. Install and set up the core [`@datadog/mobile-react-native`][3] SDK to start using Datadog Feature Flags.
```

</DatadogProvider>
```

> After completing this setup, your app should be ready for flag evaluation with OpenFeature.

Reviewer: ```suggestion
After completing this setup, your app is ready for flag evaluation with OpenFeature.
```
